src/fprime_gds/flask/flask_uploads.py

```diff
@@ -191,23 +191,20 @@ def config_for_set(uset, app, defaults=None):
     if defaults is None:
         defaults = dict(dest=None, url=None)
 
-    allow_extns = tuple(config.get(prefix + "ALLOW", ()))
-    deny_extns = tuple(config.get(prefix + "DENY", ()))
-    destination = config.get(prefix + "DEST")
-    base_url = config.get(prefix + "URL")
-
-    if destination is None:
-        # the upload set's destination wasn't given
-        if uset.default_dest:
-            # use the "default_dest" callable
-            destination = uset.default_dest(app)
-        if destination is None:  # still
-            # use the default dest from the config
-            if defaults["dest"] is not None:
-                using_defaults = True
-                destination = os.path.join(defaults["dest"], uset.name)
-            else:
-                 raise RuntimeError(f"no destination for set {uset.name}")
+    allow_extns = tuple(config.get(f"{prefix}ALLOW", ()))
```

Comment:
This file is third-part code. Perhaps we should leave it as-is to maintain an upgrade path.

Reply:
Done. My VSCode indent-rainbow plugin spotted an indentation flaw so I took the liberty to fix it.

Reply:
That is ok!